packages/contracts/package.json

```diff
@@ -12,9 +12,11 @@
     "help": "hardhat help",
     "clean": "hardhat clean",
     "lint": "solhint contracts/{*,**/*}.sol",
-    "x": "hardhat test ./test/SmartInvoiceFactory.js",
-    "y": "hardhat test ./test/SmartInvoiceEscrow.js",
-    "deploy-local": "hardhat run scripts/deploy-factory-local.js --network localhost",
+    "test-factory": "hardhat test ./test/SmartInvoiceFactory.js",
+    "test-escrow": "hardhat test ./test/SmartInvoiceEscrow.js",
+    "escrow-add-goerli": "hardhat run scripts/addEscrow.js --network goerli",
```

Comment:
I don't see an addEscrow.js file anywhere. Assuming you mean add-escrow.js instead.

Reply:
ah, good catch. that was the previous name.